n0s3p4ss/sniffer_switcher_http_status_based.py

```diff
@@ -42,7 +47,33 @@ def apply(self, attack_surface):
 class WebServerAnalyser:
 
     def apply(self, attack_surface):
-        return ServerHeaderAnalyser().apply(attack_surface)
+        report = ServerHeaderAnalyser().apply(attack_surface)
+        headers = attack_surface.http_response.headers
```

Comment:
This is not a problem but I think it's safer do:

```suggestion
        headers = attack_surface.get('http_response', {}).get('headers', {})
```

It's important set a default value to `headers` because `sec_headers_obtainer.py` has no validation on `headers` :trollface: .

Reply:
Okay :trollface: